frontend/providers/imagehub/Makefile

```diff
@@ -0,0 +1,48 @@
+
+SERVICE_NAME=zhujingyang/sealos-image-hub
```

Comment:
`zhujingyang/sealos-image-hub`  Is the image url here incorrect? It should be placed in the public repository.

Reply:
It will become a public warehouse later